flytekit/configuration/file.py

```diff
@@ -120,15 +180,33 @@ def _get_from_legacy(self, c: LegacyConfigEntry) -> typing.Any:
 
         return self._legacy_config.get(c.section, c.option)
 
-    def get(self, c: typing.Union[LegacyConfigEntry]) -> typing.Any:
+    def _get_from_yaml(self, c: YamlConfigEntry) -> typing.Any:
+        keys = c.switch.split(".")  # flytectl switches are dot delimited
```

Comment:
you can use jsonpath reader - https://linuxhint.com/jsonpath-python/. But, I guess instead of one more dependency, this might be good